packages/apps/docs/src/components/PageHelpful/useModal.tsx

```diff
@@ -0,0 +1,75 @@
+import {
```

Comment:
maybe make the name more specific for the pagehelp thing. because this hook is specific